### ISSUE TITLE
Reach modifier stuck after updating from SevTech 3.0.8 to 3.1.1

### ISSUE THREAD

**Issue by PownerTech** [HellFirePvP/AstralSorcery#1149]
Hello HellFirePvP, first of all, thank you for this awesome mod, I almost forgot I was playing SevTech since I got lost doing Astral Sorcery stuff for days !

Now for the issue. Today I updated my SevTech multiplayer server from 3.0.8 to 3.1.1. The update came with version 1.10.11 of Astral Sorcery (awesome new block designs by the way), this newer version of Astral Sorcery came with the new perk tree and I think that caused the issue.

Before updating I was attuned to Aevitas, so I had the perk that increased my reached by 3.5 blocks. After updating I lost every perk but I kept the reach increase, it's getting quite broken with other reach increases from the perk tree.

I tried using a Shifting Star, rebooting my client and the server. I'm the owner of the server so I have access to the files.

I downloaded the server files to play on a local environment in order to try and pinpoint the file containing the modifier. But no luck, I can't seem to find where the modifier is stored. I deleted the AstralSorceryData and playerdata folders to see if it was stored within one of those.

In my local world I tried again using a Shifting Star, attuning myself with command, resetting all my progress within Astral Sorcery but none of those worked, I always keep the extra 3.5 blocks of reach.

Do you have any idea how to get this fixed ? If the information is stored somewhere I can access the file and modify it. Thanks for your time and again awesome job on this mod.

**Comment by Doomgull** [comment]
I would have expected it in playerdata because it is an attribute modifier change.

**Comment by PownerTech** [comment]
My error was to test deleting files while loading the server data in a single player world. The files are not stored in the same way, so even when deleting my player data it would come back unaffected.

I fixed it by storing my inventory in a chest, stopping the server, deleting my player data (.invo, .astral, .astralback, .cyclicinvo and .dat files) on the server then restarting. Only deleting the .dat file might fix the issue but to avoid conflicting data I deleted everything about my player but my achievements and progress tied to SevTech. I only lost my Astral Sorcery progress while doing so and it fixed the issue.

Thanks a lot Doomgull for the help over on discord ! Keep up the good work on this awesome mod :)
